### ISSUE TITLE
A11y_MicrosoftCollaborate_AvailableEngagements_ScreenReader: The join links are not descriptive to inform what link / webpage will be opened.

### ISSUE THREAD

**Issue by fsteffi** [nvaccess/nvda#10364]
**GitHubTags:** #A11yTCS; #A11yMAS;#DesktopWeb; #Win10; #A11y_MicrosoftCollaborate; #MAS2.4.4; #Narrator; #NVDA; #Micorsoft Collaborate; #Edge; #Chrome

**Environment Details:** 
Windows Version: Windows10
Browser Version: Microsoft Edge 44.18875.1000.0, Chrome Version 73.0.3683.103
MAS violated: MAS2.4.4

**Steps to Reproduce:**
1. Open the Microsoft Collaborate application using provided url.
2. Append URL, then Navigate to Collaborate button and expand it --> Select "Overview menu item" --> Click on "Go to Engagements page link" --> Click on "Join button"
3. Start screen reader.
4. Navigate to go to the ‘Join’ links in the table

**Actual Result:**
When focus moves ‘Join’ link the screen reader only announces ‘Join Engagement’.

**Expected Result:**
When focus moves ‘Join’ link the screen reader should announces the page name along with ‘Join Engagement’ or give more description about the page that it will open.

Similar issue observed with Chrome + NVDA.

**User Impact:**
User will not be able to understand purpose of the link hence the link needs to be more descriptive.
[#21456800 A11y_MicrosoftCollaborate_AvailableEngagements_ScreenReader.pptx](https://github.com/nvaccess/nvda/files/3710639/21456800.A11y_MicrosoftCollaborate_AvailableEngagements_ScreenReader.pptx)

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
